build.sbt

```diff
@@ -53,11 +53,15 @@ lazy val hat = project
     libraryDependencies := (buildEnv.value match {
           case BuildEnv.Developement | BuildEnv.Test =>
             libraryDependencies.value ++ Seq(
-                  Library.Play.specs2,
-                  Library.Specs2.core,
-                  Library.Specs2.matcherExtra,
-                  Library.Specs2.mock,
-                  Library.Play.Silhouette.silhouetteTestkit
+                  Library.Play.Silhouette.silhouetteTestkit,
+                  Library.Test.scalatest,
```

Comment:
Which of these are bought in my testCommon ? Do we need to repeat dependencies that are bought in transitively (reduces version clashes, makes upgrading easier etc)

Reply:
Good question, I will need to investigate.